gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -1484,6 +1490,10 @@ function SWEP:Initialize()
         self:InitializeCustomModels()
     end
 
+    if self.SetClipOnBuy then
+        self:SetClip1(self.ClipOnBuy or self.Primary.DefaultClip)
+    end
+
```

Comment:
Arent those also called for weapons on the floor? Is that intended?

Reply:
Yes. It wasn't when I came up with the idea, but I changed my mind. Maybe that means that I should change the variable name. Any good suggestions? Maybe configurableClip